dockstore-integration-testing/src/test/java/io/dockstore/client/cli/WorkflowIT.java

```diff
@@ -1761,6 +1763,53 @@ public void testWorkflowVersionAliasesAreReturned() throws ApiException {
 
     }
 
+    @Test
+    public void testGitReferenceParsing() {
```

Comment:
I would have put this in a unit test in dockstore-webservice, e.g., in GitHelperTest.java, that way you can run test tests locally and more quickly.